packages/@stimulus/core/src/controller.ts

```diff
@@ -3,16 +3,22 @@ import { Context } from "./context"
 import { DataMap } from "./data_map"
 import { Scope } from "./scope"
 import { TargetSet } from "./target_set"
+import { defineTargetProperties } from "./target_properties"
 
 export interface ControllerConstructor {
-  targets: string[]
+  import()
   new(context: Context): Controller
 }
 
 export class Controller {
   static targets: string[] = []
+
   readonly context: Context
 
+  static import() {
```

Comment:
Looking at this externally, I'd expect `import()` to take argument (and import it). How about `setup()` or `prepare()`?

Reply:
I picked `import` to match the existing language around `importDefinition`, and for parity with the present-tense lifecycle method names (`initialize`, `connect`, `disconnect`). Fair points though. Of the two I think I prefer `setup`.

Reply:
What do you think of the name `bless`?

In Perl, you [`bless` a hash reference to associate it with a class](https://www.perl.com/pub/1999/09/refererents.html/#rule-3-to-create-an-object-bless-a-referent).

This [Stack Overflow answer](https://stackoverflow.com/a/392194) explains it succinctly:

> The effect bless-ing has is that it allows you to apply special syntax to the blessed reference.

It seems fair to say that what we are doing here is applying special properties to a “blessed” controller class.

Reply:
Love that. #blessed

Reply:
🙏